app/services/recurring_contribution/subscriptions/create_pagarme.rb

```diff
@@ -22,7 +22,7 @@ def default_attributes
     {
       plan: ::Pagarme::API.find_plan(@plan_id),
       payment_method: @payment_method,
-      postback_url: "http://test.com/postback",
+      postback_url: Rails.application.routes.url_helpers.subscription_status_update_url,
```

Comment:
Yep! Something like:

```ruby
def postback_url
   Rails.application.routes.url_helpers.subscription_status_update_url
end
```

and then use it like this: `postback_url: postback_url`, it'll make your method easier to read.